patches/api/0274-Add-missing-effects.patch

```diff
@@ -28,49 +28,131 @@ index d0c812c72967469122f7f276ced31cfb0e9482dc..0f598fa0560d14b44dcc65205364870c
 +    /**
 +     * The sound of phantom's bites
```

Comment:
I guess for the sake of consistency, this one is missing the `@deprecated use @{link #PHANTOM_BITE}` comment.